src/main/scala/org/broadinstitute/dsde/workbench/sam/service/StatusService.scala

```diff
@@ -35,7 +36,7 @@ class StatusService(
 
   private def checkOpenDJ(groupToLoad: WorkbenchGroupName): IO[SubsystemStatus] = {
     logger.info("checking opendj connection")
-    directoryDAO.loadGroupEmail(groupToLoad).map {
+    directoryDAO.loadGroupEmail(groupToLoad, SamRequestContext(None)).map {
```

Comment:
can status checks have request contexts?

Reply:
I could add that in (with no trace) if that's something we want. After discussing with Doug, the Status and Swagger routes were left without traces, but we could add in the plumbing for the context to one or both.